packages/@sfcc-bff/productapi/src/api/models/priceHelpers.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
why is priceHelpers put in the models folder? Maybe it should follow the format we use for other models. And then we can use this model like we use Image model.

Reply:
same as above

Reply:
yes but it is not a model, so maybe we should create a helper folder. or make this into a Price.js model fully. Image.js is a small model too.

Reply:
okay, this file will be moved to the schema folder.